packages/avatar-creator/src/scripts/avatar-loader.ts

```diff
@@ -140,19 +140,93 @@ export class AvatarLoader extends EventHandler {
       this.entity.addChild(entity);
     }
 
-    this.loadAnimation();
+    // list of animations
+    const animations = [
+      ["appear", "spawn_and_wave.glb"],
+      ["clap", "clap.glb"],
+      ["wave", "pick_me.glb"],
+      ["thumbsDown", "thumbs_down.glb"],
+      ["thumbsUp", "thumbs_up.glb"],
+    ];
+
+    // add animation data to anim-graph
+    for (const item of animations) {
+      const layer = animGraphData.layers[0];
+      const name = item[0];
+
+      // state
+      layer.states.push({
+        name,
+        speed: 1,
+        loop: false,
+        defaultState: false,
+      });
+
+      // transition in
+      layer.transitions.push({
+        from: "ANY",
+        to: name,
+        exitTime: 0,
+        time: 0.1,
+        interruptionSource: "NONE",
+        edgeType: 1,
+        conditions: [
+          {
+            parameterName: name,
+            predicate: "EQUAL_TO",
+            value: true,
+          },
+        ],
+      });
+
+      // transition out
+      layer.transitions.push({
+        from: name,
+        to: "Idle",
+        exitTime: 0.9,
+        interruptionSource: "NONE",
+        edgeType: 1,
+        conditions: {},
+        time: 0.1,
+      });
+
+      // trigger
+      animGraphData.parameters[name] = {
+        name,
+        type: "TRIGGER",
+        value: false,
+      };
+    }
+
+    // load anim-graph
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    entity.anim!.loadStateGraph(animGraphData);
+
+    // load default idle animation
+    this.loadAnimation("Idle", "idle.glb");
+
+    // load additional animations
+    for (const item of animations) this.loadAnimation(item[0], item[1]);
+
+    // trigger "appear" animation by default
+    entity.anim?.setTrigger("appear");
+
+    // listen to global event on app for animation triggers
+    this.app.on("anim", (name) => {
+      // eslint-disable-next-line
+      if (!entity.anim?.parameters.hasOwnProperty(name)) return;
+      entity.anim?.setTrigger(name, true);
+    });
   }
 
   /**
    * @private
    */
-  loadAnimation() {
-    const fileName = "idle.glb";
-
-    this.assetAnimIdle = new Asset(
+  loadAnimation(name: string, fileName: string) {
+    const asset: Asset = new Asset(
       fileName,
       "container",
-      { url: idleAnimationGLB, filename: fileName },
+      { url: `/anim/${fileName}`, filename: fileName },
```

Comment:
It probably should have been the case before, but it's more obvious now - can we make the animation file paths configurable rather than assumed to be relative paths from the page?